docs/_config.yml

```diff
@@ -26,7 +26,7 @@ anchor_js_targets: [h1, h2, h3, h4, h5, h6]
 # Enables Private Eye functionality.
 # See https://github.com/18F/private-eye for more information.
 # Private Eye configuration is in assets/js/main.js
-private_eye: true
+private_eye: false
```

Comment:
Since we don't use this, it was generating errors such as:

``[2021-04-28 22:49:24] ERROR `/assets/js/vendor/private_eye.js' not found.``